### ISSUE TITLE
Provider not being used?

### ISSUE THREAD

**Issue by mylar3user** [mylar3/mylar3#381]
Just installed Mylar3, and it's working nicely so far. But unless I'm reading the logs incorrectly, it's only using the experimental search and not my other provider (dog). I've tried checking and unchecking it, starting and restarting Mylar. I had another newznab provider that I tried to remove to avoid confusion, but it's still the same. I also tried adjusting the provider order in config.ini and while it looks like it has it there, it's not actually using it, at least according to the logs.

[carepackage.zip](https://github.com/mylar3/mylar3/files/4848507/carepackage.zip)

OS: Raspbian (DietPi)
Branch: Python3-dev (latest pull)

**Comment by barbequesauce** [comment]
It's definitely hitting Dog's RSS... it doesnt seem to be doing API calls though, not sure why.

I assume you saved your config and restarted prior to initiating search?

**Comment by mylar3user** [comment]
Yes, thanks.

**Comment by barbequesauce** [comment]
OK - let's see what @evilhero comes up with.

**Comment by mylar3user** [comment]
[carepackage-2.zip](https://github.com/mylar3/mylar3/files/4861506/carepackage-2.zip)

Just as a test, I tried adding Cat, and it immediately worked. I've tried removing the provider order (when shut down) and rebooting, and it didn't work. For some reason Dog isn't working with the API. I checked their site and don't see any API issues. Just weird. Still looking at old fixes in the hopes of finding a similar issue.

**Comment by evilhero** [comment]
It looks like the dognzb RSS isn't working anymore as it should.

So basically, their staff have said that all requests that use the API have to thru their API url, not their web url. If it goes thru the web url, it will be rejected with a message and I guess temporarily blocked in some fashion. 

When Mylar searches it uses the API url as it should, however when it polls the RSS it uses the web url which is supposed to be used for RSS queries only. However, the apikey is attached to the request (as per their website) - so I dunno if they're just seeing if any requests going to the web url with an apikey and automatically reject it or something else is happening. 

So basically, when Mylar goes to poll the RSS and the respective web url, it's being rejected because it's not going to their API url.

Not a big deal, we can switch the RSS url out for the respective API url when doing RSS searches - only difference is that it will cost an API hit per search as opposed to the previous RSS search which was 'hit' free (not for every indexer, but for dog at least).

**Comment by mylar3user** [comment]
Ok, nothing I can do on my end until the swap happens in the code, correct? If so, I’ll be happy to test when it is. Thanks!

**Comment by evilhero** [comment]
Yep, once it gets merged (today at some point, you'll get a notification probably cause I tagged this issue #) - just update your copy and then you should be good to go.

**Comment by mylar3user** [comment]
[carepackage-3.zip](https://github.com/mylar3/mylar3/files/4880817/carepackage-3.zip)

Just updated and checked the logs, and it's looking good so far. Will keep watching later in the week too. Thank you very much!

**Comment by mylar3user** [comment]
This appears to be working, so I'll close this issue. Thanks again for your help!
